footer.php

```diff
@@ -2,33 +2,28 @@
 /**
  * The template for displaying the footer
  *
- * Contains the closing of the #content div and all content after.
- *
  * @link https://developer.wordpress.org/themes/basics/template-files/#template-partials
  *
  * @package AMPConf
  */
 
 ?>
 
-	</div><!-- #content -->
+<footer class="wrap wrap--full-width wrap--footer">
+	<nav class="wrap__item wrap__item--footer-menu">
+		<?php
+			wp_nav_menu(
+				array(
+					'theme_location' => 'footer',
+					'menu_id'        => false,
+					'depth'          => 1,
+				)
+			);
+		?>
+	</nav>
 
-	<footer id="colophon" class="site-footer">
-		<div class="site-info">
-			<a href="<?php echo esc_url( __( 'https://wordpress.org/', 'ampconf' ) ); ?>">
-								<?php
-								/* translators: %s: CMS name, i.e. WordPress. */
-								printf( esc_html__( 'Proudly powered by %s', 'ampconf' ), 'WordPress' );
-			?>
-			</a>
-			<span class="sep"> | </span>
-			<?php
-				/* translators: 1: Theme name, 2: Theme author. */
-				printf( esc_html__( 'Theme: %1$s by %2$s.', 'ampconf' ), 'ampconf', '<a href="http://underscores.me/">Me</a>' );
-			?>
-		</div><!-- .site-info -->
-	</footer><!-- #colophon -->
-</div><!-- #page -->
+	<p class="wrap__item wrap__item--footer-copyright">&copy; <?php echo esc_html( date( 'Y' ) ); ?> <a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></p>
```

Comment:
I think `&copy;` is universal but I think we should change it to `<?php printf( '&copy; %s', esc_html( date( 'Y' ) ) ); ?>` just to be safe.